### ISSUE TITLE
[BUG] UI shows weird cursor behaviour when using wacom tablet

### ISSUE THREAD

**Issue by PrzemoF** [obsproject/obs-studio#2763]
## Platform
Operating system and version: Fedora Linux, ver 31 installed from official repository (rpm)
OBS Studio version: 25.0.4 (64bit)

## Expected Behavior
Normal UI cursor behaviour. When I click on menu it should open, when I click on button it should be activated.

## Current Behavior
I use wacom tablet instead of a normal mouse. With normal mouse UI works as expected. With tablet
cursor activities (clicking, highlighting items) partially works, doesn't work or it's shifted.
I can't open menu bar with clicking. I can do it with mouse and then I can use tablet to highlight and activate menu items. On the main screen highlighting items is happening about 2 cm below the actual position of the cursor. A video showing the behaviour: https://peertube.social/videos/watch/8c0cd4e1-8c57-450a-8d50-d894b6ae59dd
Watch position of the cursor and position of the highlighted item. I started/stopped toe video by clicking on frame of the Controls panel

Log [2020-04-20 09-27-28.txt](https://github.com/obsproject/obs-studio/files/4502168/2020-04-20.09-27-28.txt)

## Steps to Reproduce
1. Open OBS Studio
2. Try to do anything using wacom table, like opening menu bar or starting recording

I use normally use gnome in (xorg) mode, but wayland mode shows the same problem. OBS is the only application showing that behaviour - I use FreeCAD, GIMP, Inkscape and others (both GTK and Qt based), so the problem is most likely in the way OBS treats input data from xorg system.

**Comment by ChrBohm** [comment]
I have the same problem. Ubuntu Mate 20.4.

**Comment by gwillegers** [comment]
I have the same problem. OS is Linux Mint 20 Ulyana (based on Ubuntu 20.04). Kernel 5.4.0-52-generic, 64 bit. All updates installed.

OBS Studio installed from repositories, version 25.0.3.
OBS Studio responds to the keyboard, but not to the pen or mouse of the Wacom Intuos 4. Every else is working fine, it is just OBS Studio that has this problem. As a test I plugged in an USB mouse, which is working fine in OBS Studio.

I added the PPA for OBS Studio, as described in [the installation instructions](https://obsproject.com/wiki/install-instructions#linuxl), under Ubuntu. I then uninstalled 25.0.3, and installed 26.0.2 from the PPA. This did not solve the problem.
